packages/jest-cli/src/runTest.js

```diff
@@ -19,11 +19,34 @@ const {
   NullConsole,
   setGlobal,
 } = require('jest-util');
+
+const fs = require('fs');
+const docblock = require('jest-docblock');
+const {getTestEnvironment} = require('jest-config');
```

Comment:
Would love to use a lint plugin to enforce the current rules we have. Mind sending a separate PR for that? :)